discordbot/OverworldDiscordBot.py

```diff
@@ -0,0 +1,282 @@
+                            image_binary.seek(0)
+                            # save to a discord file
+                            file = discord.File(fp=image_binary, filename='image.png')
+                        # set embed thumbnail to the attached discord file
+                        embed.set_thumbnail(url="attachment://image.png")
+                        
+                        # send a message in channel start was called with the infomation of the pokemon
+                        # asyncio.run_coroutine_threadsafe(ctx.send(embed=embed,file=file),self.loop)
+                        for channel_id in _channels_to_notify:
+                            asyncio.run_coroutine_threadsafe(self.send_discord_event(embed, file, channel_id),self.loop)
+                # print a line to show that new pokemon have just been read
+                _message = "{} Pokemon Observed".format(len(pkms))
+                asyncio.run_coroutine_threadsafe(self.send_discord_msg(_message, Channels.NotificationChannelForInfo),self.loop)
+            # give the thread a break
+            self.reader.pause(0.3)
+
+def is_pkm_shiny(pkm):
+    return pkm.shinyType != 0
+
+def is_pkm_brilliant(pkm):
+    return pkm.brilliant
+
+def is_pkm_marked(pkm):
+    return pkm.mark > 52 and pkm.mark < 98 and pkm.mark != 69
+
+def is_pkm_rare(pkm):
+    return pkm.mark == 69
+
+def get_pkm_species_string(pkm):
+    return Util.STRINGS.species[pkm.species] + (('-' + str(pkm.altForm)) if pkm.altForm > 0 else '')
+
+def get_pkm_mark_string(pkm):
+    return 'No Mark' if pkm.mark == 255 else pkm.Ribbons[pkm.mark]
+
+# function that formats the pkm into a title string and a description string for discord embeds
+def pkm_format(pkm, context):
+    # get strings from pkm to use
+    # 1/2/H
+    ability_string = str(pkm.ability) if pkm.ability < 4 else 'H'
+    # Brilliant/Not Brilliant
+    brilliant_string = 'Brilliant' if pkm.brilliant else 'Not Brilliant'
+    # ♂/♀/-
+    gender_emoji = Util.GenderSymbol[pkm.gender-1]
+    # X/X/X/X/X/X
+    ivs_string = '/'.join(str(x) for x in pkm.ivs)
+    # pulls an emoji from the current server with the id of the mark as its name
+    # will throw an error unless you have all the marks as emojis
+    mark_emoji = (str(get(context.message.guild.emojis, name=str(pkm.mark))) + ' ') if pkm.mark != 255 else ''
+    # No Mark/Mark Name
+    mark_string = get_pkm_mark_string(pkm)
+    # Nature string (Jolly,Adamant,Timid,etc.)
+    nature_string = Util.STRINGS.natures[pkm.nature]
+    #  /⋆/◇
+    shiny_string = '' if pkm.shinyType == 0 else '⋆ ' if pkm.shinyType == 1 else '◇ '
+    # Species-Form Number (Articuno-1 for GCuno)
+    species_string = get_pkm_species_string(pkm)
+
+    # format of the title
+    title = f"{mark_emoji}{gender_emoji} {shiny_string} {species_string}"
+    # format of the description
+    description = f"{nature_string} {ability_string}\n{ivs_string}\n{mark_string}\n{brilliant_string}"
+
+    return title, description
```

Comment:
Note for future: have a way to make formatting easier for the user, assume little knowledge of python

Reply:
i would say it wouldn't be high priority, but some users might wish to have their embeds look different and it would be a nice option

Reply:
Added to the readme!